src/plugins/js-eval/jsEvalPlugin.js

```diff
@@ -1,58 +1,19 @@
-const parseEvalMessage = require('./parseEvalMessage');
-const runDockerContainer = require('./runDockerContainer');
-const formatEvalResponse = require('./formatEvalResponse');
-// const parseCode = require('./parseCode');
-// const annotateCode = require('./annotateCode');
-const parseOutput = require('./parseOutput');
+const { exec } = require('child_process');
 
-const jsEvalPlugin = ({ mentionUser, respond, respondWithMention, handling, message }) => {
-  const initialParams = parseEvalMessage(message);
-  if (!initialParams) return;
-  handling(initialParams);
+const jsEvalPlugin = ({ mentionUser, respond, respondWithMention, message }) => {
+  if (!message.startsWith('n>')) return;
+  const code = message.slice(2);
 
-  // let ast = null;
-  try {
-    // parseCode(initialParams.code);
-    // ast = parseCode(initialParams.code);
-  } catch (e) {
-    respondWithMention(`Failed to parse code: ${e.message}`);
-    return;
-  }
+  const respondWith = mentionUser ? respondWithMention : respond;
 
-  // const annotated = annotateCode(ast);
-  // const params = { ...initialParams, code: annotated };
-  const params = { ...initialParams, code: initialParams.code };
+  const child = exec('docker run --rm -i devsnek/js-eval', { timeout: 5000 }, (err, stdout = '') => {
```

Comment:
The options from [here](https://github.com/brigand/jellobot/blob/master/src/plugins/js-eval/runDockerContainer.js#L8) need to be provided.